libjwt/jwt.c

```diff
@@ -1514,3 +1517,53 @@ unsigned int jwt_validate(jwt_t *jwt, jwt_valid_t *jwt_valid)
 
 	return jwt_valid->status;
 }
+
+typedef struct {
+	int error;
+	char *str;
+} jwt_exception_dict_t;
+
+static jwt_exception_dict_t jwt_exceptions[] = {
+	/* { JWT_VALIDATION_SUCCESS, "SUCCESS" }, */
+	{ JWT_VALIDATION_ERROR, "general failures" },
+	{ JWT_VALIDATION_ALG_MISMATCH, "algorithm mismatch" },
+	{ JWT_VALIDATION_EXPIRED, "token expired" },
+	{ JWT_VALIDATION_TOO_NEW, "token future dated" },
+	{ JWT_VALIDATION_ISS_MISMATCH, "issuer mismatch" },
+	{ JWT_VALIDATION_ISS_MISMATCH, "subject mismatch" },
+	{ JWT_VALIDATION_AUD_MISMATCH, "audience mismatch" },
+	{ JWT_VALIDATION_GRANT_MISSING, "grant missing" },
+	{ JWT_VALIDATION_GRANT_MISMATCH, "grant mismatch" },
+};
+
+char *jwt_exception_str(unsigned int exceptions)
+{
+	int rc;
+	char *str = NULL;
+
+	if (exceptions == JWT_VALIDATION_SUCCESS) {
+		if ((rc = __append_str(&str, "success")))
+			goto fail;
+		return str;
+	}
+
+	for (int i = 0; i < ARRAY_SIZE(jwt_exceptions); i++)
```

Comment:
I'm getting nit picky on this one, but it's a style issue. Please use braces `{}` around the code in the for-loop.

Reply:
done